src/main/java/com/epam/aidial/core/data/FileMetadataBase.java

```diff
@@ -9,6 +9,8 @@
 @NoArgsConstructor
 public abstract class FileMetadataBase {
     private String name;
-    private String path;
+    private String parentPath;
```

Comment:
Can we rename to folder?

Reply:
it's not a folder, it's a path, like folder1/folder2/folder3